File: NetworkRequest/app/src/main/java/ai/tomorrow/networkrequest/MyApplication.java
package ai.tomorrow.networkrequest;

import android.app.Application;

import ai.tomorrow.base.NetManager;

public class MyApplication extends Application {

    private static final String TAG = "MyApplication";

    private NetManager mNetManager;

    @Override
    public void onCreate() {
        super.onCreate();

        mNetManager = new NetManager(this);
    }

    public NetManager getNetManager() {
        return mNetManager;
    }
}

File: protobuf/githubApi_kotlin/app/src/main/java/com/example/android_starter/base/MyApplication.kt
package com.example.android_starter.base

import android.app.Application
import com.example.base.BaseManager

/**
 *  Created by Zhaolong Zhong on 08/11/2019.
 */

class MyApplication: Application() {

    lateinit var baseManager: BaseManager

    override fun onCreate() {
        super.onCreate()
        baseManager = BaseManager(this)

        init()
    }

    private fun init() {
        // Stub
    }
}